test/sqlOutputContentProvider.test.ts

```diff
@@ -1,105 +1,160 @@
-    //     let uri = contentProvider.updateContent(messages, results);
-    //     let url = 'http://localhost:' + port + '/' + Interfaces.ContentTypes[Interfaces.ContentType.Columns] + '?id=0&uri=' + uri;
-    //     return request.get(url, function(err, res, body){
-    //         assert.equal(res.statusCode, 200);
-    //         assert.equal(body, JSON.stringify(results[0].columns));
-    //     });
-    // });
-
-    // test("Correctly Delievers Rows", () => {
-    //     let uri = contentProvider.updateContent(messages, results);
-    //     let url = 'http://localhost:' + port + '/' + Interfaces.ContentTypes[Interfaces.ContentType.Rows] + '?id=0&uri=' + uri;
-    //     return request.get(url,(err, res, body) => {
-    //         assert.equal(res.statusCode, 200);
-    //         assert.equal(body, JSON.stringify(results[0].rows));
-    //     });
-    // });
-// });
+import { SqlOutputContentProvider } from '../src/models/SqlOutputContentProvider';
+import VscodeWrapper from '../src/controllers/vscodeWrapper';
+import StatusView from '../src/views/statusView';
+import * as stubs from './stubs';
+import Constants = require('../src/models/constants');
+import vscode = require('vscode');
+import * as TypeMoq from 'typemoq';
+import assert = require('assert');
+
+suite('SqlOutputProvider Tests', () => {
+    let vscodeWrapper: TypeMoq.Mock<VscodeWrapper>;
+    let contentProvider: SqlOutputContentProvider;
+    let context: TypeMoq.Mock<vscode.ExtensionContext>;
+    let statusView: TypeMoq.Mock<StatusView>;
+
+    setup(() => {
+        vscodeWrapper = TypeMoq.Mock.ofType(VscodeWrapper);
+        context = TypeMoq.Mock.ofType(stubs.TestExtensionContext);
+        context.object.extensionPath = '';
+        statusView = TypeMoq.Mock.ofType(StatusView);
+        contentProvider = new SqlOutputContentProvider(context.object, statusView.object);
+        contentProvider.setVscodeWrapper(vscodeWrapper.object);
+
+    });
+
+    test('Correctly outputs the new result pane view column', done => {
+        function setSplitPaneSelectionConfig(value: string): void {
+            let configResult: {[key: string]: any} = {};
+            configResult[Constants.configSplitPaneSelection] = value;
+            let config = stubs.createWorkspaceConfiguration(configResult);
+            vscodeWrapper.setup(x => x.getConfiguration(TypeMoq.It.isAny()))
+            .returns(x => {
+                return config;
+            });
+        }
+
+        function setCurrentEditorColumn(column: number): void {
+            vscodeWrapper.setup(x => x.activeTextEditor)
+            .returns(x => {
+                let editor: vscode.TextEditor = new stubs.TestTextEditor();
+                editor.viewColumn = column;
+                return editor;
+            });
+        }
+
+        class Case {
+            position: number;
+            config: string;
+            expectedColumn: number;
+        }
+
+        let cases: Case[] = [
```

Comment:
It would be nice if each of these was a test case. See https://mochajs.org/#dynamically-generating-tests for how to easily convert this (basically define cases at the `suite` level and in the foreach define a test.